plugins/org.jkiss.dbeaver.ext.gbase8s/src/org/jkiss/dbeaver/ext/gbase8s/GBase8sUtils.java

```diff
@@ -67,16 +66,25 @@ public static List<String> getSource(DBRProgressMonitor monitor, String sqlState
     }
 
     public static boolean isOracleSqlMode(DBPDataSourceContainer container) {
-        final DBPDriver driver = container.getDriver();
-        Map<String, Object> cprops = driver.getConnectionProperties();
-        for (Map.Entry<String, Object> entry : cprops.entrySet()) {
-            if (Objects.equals(entry.getKey().toLowerCase(), "sqlmode")) {
-                if (Objects.equals(entry.getValue().toString().trim().toLowerCase(), "oracle")) {
-                    return true;
-                }
+        DBPConnectionConfiguration configuration = container.getConnectionConfiguration();
+        String sqlMode = getSqlModeValue(configuration.getProperties());
+        if (sqlMode != null) {
+            return GBase8sConstants.JDBC_SQL_MODE_ORACLE.equalsIgnoreCase(sqlMode);
+        }
+        return GBase8sConstants.JDBC_SQL_MODE_ORACLE
+                .equalsIgnoreCase(getSqlModeValue(container.getDriver().getConnectionProperties()));
+    }
+
+    private static String getSqlModeValue(Map<String, ?> properties) {
+        if (properties == null || properties.isEmpty()) {
+            return null;
+        }
+        for (Map.Entry<String, ?> entry : properties.entrySet()) {
+            if (GBase8sConstants.JDBC_SQL_MODE.equalsIgnoreCase(entry.getKey())) {
+                return entry.getValue() != null ? entry.getValue().toString().trim() : null;
```

Comment:
Use **org.jkiss.utils.CommonUtils#trim** instead of  
`entry.getValue() != null ? entry.getValue().toString().trim() : null;`

Reply:
It expects a `String`